From ef92d723e344c7f801616b07e23cf5154160b666 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 23 Mar 2018 14:19:13 +0800
Subject: [PATCH] arm64: rk3308_linux_defconfig: disable CPU_IDLE_GOV_LADDER

Ladder governor is for periodic timer tick, we use menu governor.

Change-Id: Iffb797a801563f6745238a26b2582ea57d6ab0a5
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 arch/arm64/configs/rk3308_linux_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/configs/rk3308_linux_defconfig b/arch/arm64/configs/rk3308_linux_defconfig
index ff0dc60bc3b8..6d11859127a2 100644
--- a/arch/arm64/configs/rk3308_linux_defconfig
+++ b/arch/arm64/configs/rk3308_linux_defconfig
@@ -45,6 +45,7 @@ CONFIG_PM_DEBUG=y
 CONFIG_PM_ADVANCED_DEBUG=y
 CONFIG_WQ_POWER_EFFICIENT_DEFAULT=y
 CONFIG_CPU_IDLE=y
+# CONFIG_CPU_IDLE_GOV_LADDER is not set
 CONFIG_ARM_CPUIDLE=y
 CONFIG_CPU_FREQ=y
 CONFIG_CPU_FREQ_DEFAULT_GOV_INTERACTIVE=y
-- 
2.35.3

